cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
+fixes: 
+  - 
+    commit: b45c377f8f488955e0c7069cad3f3dd21910b071
+    note: ~
+  - 
+    commit: 92d3430f12171f16f566c9050c40feefb830a4a3
+    note: ~
+fixes_vcc_instructions: |
+    Please put the commit hash in "commit" below (see my example in
+    CVE-2011-3092.yml). Fixes and VCCs follow the same format.
+i18n: 
+  answer: false
+  note: "the vulnerability impacts displayed URL which has no variation depending on user location.\n"
   question: |
-    How was this vulnerability discovered?
-
-    Go to the bug report and read the conversation to find out how this was
-    originally found. Answer in longform below in "answer", fill in the date in
-    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
-    employee (you can tell from their email address). If it's clear that the
-    vulenrability was discovered by a contest, fill in the name there.
-
-    The automated, contest, and developer flags can be true, false, or nil.
-
-    If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
-autodiscoverable:
-  instructions: |
-    Is it plausible that a fully automated tool could have discovered
-    this? These are tools that require little knowledge of the domain,
-     e.g. automatic static analysis, compiler warnings, fuzzers.
-
-    Examples for true answers: SQL injection, XSS, buffer overflow
-
-    Examples for false: RFC violations, permissions issues, anything
-    that requires the tool to be "aware" of the project's
-    domain-specific requirements.
-
-    The answer field should be boolean. In answer_note, please explain
-    why you come to that conclusion.
-  answer_note:
-  answer:
-specification:
-  instructions: |
-    Is there mention of a violation of a specification? For example,
-    an RFC specification, a protocol specification, or a requirements
-    specification.
-
-    Be sure to check all artifacts for this: bug report, security
-    advisory, commit message, etc.
-
-    The answer field should be boolean. In answer_note, please explain
-    why you come to that conclusion.
-  answer_note:
-  answer:
-subsystem:
+      Was the feature impacted by this vulnerability about internationalization
+      (i18n)? An internationalization feature is one that enables people from all
+      over the world to use the system. This includes translations, locales,
+      typography, unicode, or various other features.
+      
+      Answer should be boolean. Write a note about how you came to the conclusions
+      you did.
+interesting_commits: 
+  commits: 
+    - 
+      commit: 0714b0f39033a51e447bc1b3b03ffee6f9228105
+      note: "This commit affects parsing of values in HTTP Header. This is not explicitly related to passwords but it could maybe have an effect on the way an entered value was interpreted by the Header, which is where the vulnerability was.\n"
```

Comment:
Wrong version of affect/effect--in this instance, it should be affect because it is being used as a verb.